hack/after-build/verify-generated-protobuf.sh

```diff
@@ -39,7 +39,7 @@ done
 # We would like to use "sudo" when running on Travis and
 # not use "sudo" when running on Jenkins.
 SUDO="sudo"
-sudo -h > /dev/null || SUDO=""
+sudo -h &> /dev/null || SUDO=""
```

Comment:
actually, we still need this for running the check locally as an unprivileged user. I can clean it up a bit, though.